overlord/snapstate/snapstate.go

```diff
@@ -70,6 +72,28 @@ const (
 
 var ErrNothingToDo = errors.New("nothing to do")
 
+var osutilCheckFreeSpace = osutil.CheckFreeSpace
+
+type ErrInsufficientSpace struct {
+	// Path is the filesystem path checked for available disk space
```

Comment:
I would call this `MountPoint` - if the filesystem has a unique mount point, or instead flip the switch around to a device path. I think it's hard to explain which device maps to which path, especially on core, where we have lots of mount points and bind mounts.

Reply:
Marking resolved per the comment below.